Subject: Align PMD_SECTION_MASK with CONFIG_ARM64_VA_BITS

References: bsc#1142715

Since SUSE 4.12 kernel the VA renge is increased to 48 bits.

Upstream proposed to export the VA range in vmcoreinfo but we did not backport
the patch.

Link: https://lists.infradead.org/pipermail/kexec/2019-November/023966.html
Link: https://patchwork.kernel.org/cover/11266557/

diff -ur makedumpfile-1.6.7.orig/arch/arm64.c makedumpfile-1.6.7/arch/arm64.c
--- makedumpfile-1.6.7.orig/arch/arm64.c	2020-01-15 19:09:56.000000000 +0100
+++ makedumpfile-1.6.7/arch/arm64.c	2020-03-04 14:38:02.281783810 +0100
@@ -81,7 +81,7 @@
  * Remove the highest order bits that are not a part of the
  * physical address in a section
  */
-#define PMD_SECTION_MASK	((1UL << 40) - 1)
+#define PMD_SECTION_MASK	((1UL << 48) - 1)
 
 #define PMD_TYPE_MASK		3
 #define PMD_TYPE_SECT		1
